src/main/java/com/spring/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package com.spring.onlinebookstore.dto;
+
+import jakarta.persistence.Column;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+public record CreateBookRequestDto(
+        @NotBlank
+        String title,
+        @NotBlank
+        String author,
+        @Column(nullable = false, unique = true)
```

Comment:
Use Column annotation only with fields of entity class. Also, you can validate ISBN format using annotation Pattern and some regex from google) 
```suggestion
        @NotBlank
```